pkg/ucp/store/storeutil/id.go

```diff
@@ -164,14 +164,14 @@ func NormalizeResourceID(parsed resources.ID) (resources.ID, error) {
 	return resources.ID{}, fmt.Errorf("invalid resource id: %s", parsed.String())
 }
 
-// NormalizeResourceType normalizes the resource type to be consistent between scopes and resources.
+// ConvertScopeTypeToResourceType normalizes the resource type to be consistent between scopes and resources.
 // See comments on NormalizeResourceID for full context.
```

Comment:
upd: // See comments on ConvertScopeIDToResourceID for full context.

Reply:
Fixed, thanks